account_move_locking/README.rst

```diff
@@ -19,7 +19,7 @@ In order to lock the journal entry, you need to follow this process:
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/92/9.0
+   :target: https://runbot.odoo-community.org/runbot/92/10.0
```

Comment:
You also need to update the menus above:
Accounting > Adviser > Journal Entries